lib/runners/cli.rb

```diff
@@ -104,6 +111,7 @@ def run
             writer << Schema::Result.envelope.coerce(json)
           end
         end
+        io.finalize! if io.instance_of?(Runners::IO::AwsS3)
```

Comment:
How about this? If `Runners::IO::*` classes will be added, we should fix this code, but it's easy to forget. 😅 

```suggestion
        io.finalize! if io.respond_to?(:finalize!)
```

Reply:
Looks good. Thank you!